store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10390,4 +10390,7 @@ TODO: delete them permanently from here
   <desc>Whether retention policy feature is enabled</desc>
 </attr>
 
+<attr id="4102" name="zimbraAliasListToHide" type="email" max="256" immutable="1" cardinality="multi" optionalIn="mailRecipient" since="10.1.0">
```

Comment:
Its at account level list and particularly selected aliases to be hidden will be added to the list. Expecting it would be sufficient. Please provide your suggestion.

Reply:
I got it. 256 should be enough then. Thanks.